kubetest/BUILD

```diff
@@ -18,18 +18,23 @@ go_library(
         "anywhere.go",
         "bash.go",
         "build.go",
+        "dumplogs.go",
         "e2e.go",
         "extract.go",
         "federation.go",
         "gke.go",
         "kops.go",
         "main.go",
         "node.go",
+        "nodes.go",
```

Comment:
node.go vs nodes.go... maybe have a different name? (node.go can potentially be kubelet.go maybe?)

Reply:
Fair point.  The good news here is that this first commit is actually https://github.com/kubernetes/test-infra/pull/4371/files so we can address this in isolation there (and then I can rebase this PR).

Two options I think:

* Create kubetest/pkg/kubernetes/ or kubetest/pkg/??.   This would also mean we can reuse it elsewhere.
* Rename the file.  nodes.go -> kubernetes.go?

I'm not entirely enamored with the idea of recreating the k8s API (vs just using client-go), but I don't know if we want to take a client-go dependency.

Reply:
Renamed nodes.go -> kubernetes.go in #4371.  Doing a package is tricky because we have global state / variables, so it becomes much more involved.

Happy to rename node.go -> kubelet.go, or maybe node.go -> nodedeploy.go would be clearer?

Reply:
Personally I would like to see us start putting things into their own packages so that we stop depending on global state, but that may be better left outside the scope of this PR

Reply:
The context work may help with that :-)